easybuild/easyconfigs/y/Yambo/Yambo-5.2-intel-2021b.eb

```diff
@@ -0,0 +1,82 @@
+easyblock = 'MakeCp'
+
+name = 'Yambo'
+version = '5.2'
```

Comment:
not an official 5.2 release, so use `5.2.dev-<date-of-commit>` as version?

And maybe use `%(version_major_minor)s` below